images/installer/Dockerfile.rhel7

```diff
@@ -10,46 +10,50 @@ LABEL name="openshift3/ose-ansible" \
       summary="$SUMMARY" \
       description="$DESCRIPTION" \
       url="https://github.com/openshift/openshift-ansible" \
-      io.k8s.display-name="openshift-ansible" \
-      io.k8s.description="$DESCRIPTION" \
-      io.openshift.expose-services="" \
-      io.openshift.tags="openshift,install,upgrade,ansible" \
```

Comment:
Ditto about these labels.. they are certainly not required, but I believe it doesn't hurt to have them in this image